src/tests/unit/directory-dao.test.js

```diff
@@ -0,0 +1,74 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import _ from 'lodash';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
```

Comment:
imports should be in alphabetical order based on the package/file name. If going by file names (the second section of imports, line 7-8) then you include the directory prefix i.e. the whole `'api/v2/serializers/directory-serializer'` would be included in the alphabetization. `proxyquire` is the only one out of the place here.